site/generate.sh

```diff
@@ -92,7 +92,7 @@ mkdir -p "$WWW_ROOT_DIR"
 echo "# one update site per line" > "$MAIN_DIR"/tmp/args.lst
 
 function generate {
-  echo "--key $SECRET/update-center.key --certificate $SECRET/update-center.cert --root-certificate $( dirname "$0" )/../resources/certificates/jenkins-update-center-root-ca.crt $EXTRA_ARGS $*" >> "$MAIN_DIR"/tmp/args.lst
+  echo "--key $SECRET/update-center.key --certificate $SECRET/update-center.cert --root-certificate $( dirname "$0" )/../resources/certificates/jenkins-update-center-root-ca-2.crt $EXTRA_ARGS $*" >> "$MAIN_DIR"/tmp/args.lst
```

Comment:
Where this file `./resources/certificates/jenkins-update-center-root-ca-2.crt` come from? When I generated the secrets.zip, I rename `jenkins-update-center-root-ca-2.crt` to `jenkins-update-center-root-ca.crt` as I didn't know if it would break any scripts. 

As explained in the README from [jenkins-infra/update-center](https://github.com/jenkins-infra/update-center)
`$ zip -r secret.zip update-center.* jenkins-update-center-root-ca.crt`

Reply:
It's in https://github.com/jenkins-infra/update-center2/tree/master/resources/certificates